### PR TITLE
Ask for user credentials for PSQL commands

### PR DESCRIPTION
**Pull request by AdamHawtin**
## Context
We want to be accessing the DB with user creds rather than using the service account

## What has Changed
* Add dosql script which asks for user creds
* Update write DB access script to ask for user creds
* Update aliases

## How to Test
Build and push this image with a branch tag, deploy to a project and try the dosql command. It should prompt for a username and password which it should then authenticate with.

## Links
https://trello.com/c/u6uTsczT/1212-census-rm-toolbox-db-access-to-use-personal-accounts-instead-of-service-accounts

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[census-rm-toolbox DB access to use personal accounts instead of service accounts](https://trello.com/c/u6uTsczT/1212-census-rm-toolbox-db-access-to-use-personal-accounts-instead-of-service-accounts)

File: census-rm-toolbox-deployment-dev.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: census-rm-toolbox
-        image: eu.gcr.io/census-rm-ci/rm/census-rm-toolbox:latest
+        image: eu.gcr.io/census-rm-ci/rm/census-rm-toolbox:database-user-creds
```
Comment (ryangrundy7) [inline census-rm-toolbox-deployment-dev.yml]: Remeber to change this back

Comment (AdamHawtin) [reply census-rm-toolbox-deployment-dev.yml]: Accidental commit, good spot

**Review by NickGrantX — APPROVED** [review]
Inconvenient, but good work

**Review by ryangrundy7 — APPROVED** [review]
Works in whitelodge 👍

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**